### ISSUE TITLE
[Bug] Application "stuck" in landscape after playing a video

### ISSUE THREAD

**Issue by TheFrenchGhosty** [jellyfin/SwiftFin#102]
Follow up to a discussion we had in the Matrix room:

When opening the application screen rotation will work normally.

Playing a video in the application make the device go in landscape mode, this is fine.

The problem is that if you stop playing the video, and go back to the menu the application will be "stuck" in landscape mode.

The problem is gone if you restart the application.

Device: iPhone SE 2020

**Comment by nellistc** [comment]
Also happens on iPhone 11 running iOS 14.6.

It’s not necessary to restart the app though. If you switch between [all media] and [home], you can rotate back to portrait.

In addition, if you’re in landscape before starting a video, you can rotate to portrait immediately after exiting playback.

Also noticed that some combination of steps has left the phone with the time and other indicators in landscape mode while the Home Screen is in portrait. Never noticed that before.

![7CF77ED1-D129-4BA7-BDB5-7ABBF365DEF5](https://user-images.githubusercontent.com/1773514/125761926-14c06e0c-dac2-4d2b-9c83-aacffefb3b95.jpeg)

**Comment by sambhavsaggi** [comment]
Related https://github.com/jellyfin/SwiftFin/issues/96

**Comment by PangMo5** [comment]
The issue has been fixed in 58 build. 
Can I close this issue?

**Comment by oatmealm** [comment]
Btw does it have to rotate? I think it's more intuitive to only rotate the display if the user changed the position of the device itself or via a GUI control like "maximize" view.

**Comment by PangMo5** [comment]
> Btw does it have to rotate? I think it's more intuitive to only rotate the display if the user changed the position of the device itself or via a GUI control like "maximize" view.

It makes sense because users may use 9:16 aspect video.
But this issue is already closed and it means something else. If you really need that function, please create a new issue.
